src/main/mw.c

```diff
@@ -280,7 +280,7 @@ void processRx(void)
 
     updateRSSI(currentTime);
 
-    if (feature(FEATURE_FAILSAFE)) {
+    if (feature(FEATURE_FAILSAFE) && (rcData[THROTTLE] > 1050 || !(masterConfig.failsafeConfig.disable_failsafe_on_throttle_cut))) {
```

Comment:
this does not accound for `rxfail` mechanism (stage 1 of failsafe). When signal is lost `rxfail 3 a` will be active and throttle will be driven low, effectuvely failsafe will be never activated.